zfmt/ztests/decls.yaml

```diff
@@ -36,4 +40,6 @@ outputs:
       from (
         (internal reader)
       )
-      | pass
+      | stamp({outer:this,assignee:"bob"}) => (
+        yield {...outer,assignee:assignee,ts:now()}
+      )
```

Comment:
Unless I'm mistaken, `zc -C` output has always been a valid Zed program. Is there a compelling reason to break with that here?

Reply:
You are mistaken as was I. See how we display pushdowns in filters or specify if the output in a summarize proc is partials, I'm sure there are other examples. I did of course want to make it a valid zed program but there's some difficulties in doing this, most notably in order to scoping right I'd have to figure out how to compile the op declarations in place vs compiling where they are called. While I think this is eventually where we want to go this would to require an additional pass over the dag and this isn't a trivial change. So I made the decision to ship this and worry about making zfmt -s on op decls valid zed later because I don't think this should get in the way to getting this out.